Experimental/DistributedAuthoritySample/Assets/Scripts/UI/IngameUI/CarryBoxIndicator.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Unity.Multiplayer.Samples.SocialHub.UI
+{
+    /// <summary>
+    /// Panel that shows interaction options when character carries something.
+    /// </summary>
+    public class CarryBoxIndicator : MonoBehaviour
```

Comment:
Just a comment to go through these scripts inside IngameUI folder, and remove _public_ access modifier if possible, and to use internal in its place.

Reply:
This just needs to be applied to the UIUtils class/methods and that's looking good.

Reply:
changed visiblity on UIUtils & PlayersTopUIController sorry missed those